frontend/src/api/contexts/config/ConfigContext.tsx

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useEffect, useState } from 'react'
+import { HasChildren } from '../../../util/react-types.util'
+import axios from 'axios'
+import { API_BASE_URL } from '../../../util/configs/environment.config'
+import { ConfigDto } from './types'
+
+export const ConfigContext = createContext<ConfigDto | undefined>(undefined)
+
+export const ConfigProvider = ({ children }: HasChildren) => {
+  const [config, setConfig] = useState<ConfigDto>()
+  const [loading, setLoading] = useState<boolean>(true)
+  useEffect(() => {
+    axios
+      .get<ConfigDto>(API_BASE_URL + '/api/app')
+      .then((res) => {
+        setConfig(res.data)
+      })
+      .catch(() => console.error('Theme fetch failed!'))
```

Comment:
```suggestion
      .catch((err) => console.error('[ERROR] at ConfigProvider', JSON.stringify(err, null, 2)))
```

Reply:
I would appreciate if the error would be written to the console too (No problem, if it stays in production, they don't care about it).

this JSON-ification of the error object is a nice touch I've learned from stackoverflow